docker-compose.yml

```diff
@@ -63,7 +63,6 @@ services:
         - mailcatcher
         - redis
         - createbuckets
-        - nginx
```

Comment:
Good fix thank you.

We start often our container from our makefile.
We need to adapt it in order to work correctly.
After the line 124, we should add:
```
@$(COMPOSE) up --force-recreate -d ngnix
```

https://github.com/numerique-gouv/impress/blob/e4a7ac0f3c44f90fdaf43a582c726bd26b9a4e35/Makefile#L123-L128

Reply:
There is a typo error on the service name:  `nginx`.

Reply:
@AntoLC Can you please try again now